README.md

```diff
@@ -21,3 +21,6 @@ Przykłady użycia:
 
 6. Wymuszenie użycia STDIN/STDOUT do komunikacji:
   `python validator.py --stdio zad1 python rozwiazanie.py`
+
+7. Ustawienie mnożnika dla limitów czasowych:
+  `python validator.py --timeout-multiplier 2.5 zad1 python rozwiazanie.py`
```

Comment:
To chyba zadziała tak, że jak sie da --timeout-multilier <0 to timeout sie wyłaczy (patrz linijka 1488). Może to tu dopisać?